src/main/java/lotto/domain/LottoResult.java

```diff
@@ -20,15 +33,13 @@ public double calculateYield() {
     }
 
     private Long calculateTotalReward() {
-        return rankResults.entrySet().stream()
-                .map(result -> Rank.calculateMoney(result.getKey(), result.getValue()))
+        return rankResults.entrySet().stream().map(result -> Rank.calculateMoney(result.getKey(), result.getValue()))
                 .reduce(0L, Long::sum);
     }
 
     private int calculatePurchaseMoney() {
-        return rankResults.entrySet().stream()
-                .map(Entry::getValue)
-                .reduce(0, Integer::sum) * Lotto.LOTTO_PURCHASE_MONEY;
+        return rankResults.entrySet().stream().map(Entry::getValue).reduce(0, Integer::sum)
+                * Lotto.LOTTO_PURCHASE_MONEY;
```

Comment:
```suggestion
        return rankResults.entrySet().stream()
        .mapToInt(Entry::getValue)
        .sum() * Lotto.LOTTO_PURCHASE_MONEY;
```
IntStream 스트림을 사용하면 sum() 메서드 활용이 가능해서, 더 읽기좋을 것 같아요.
그리고 메서드체이닝(.)을 행 단위로 나눠주시면 더 읽기 좋습니다 👍

Reply:
해당 부분 반영했습니다. 확실히 IntStream을 이용하는게 가독성이 좋네요